libnfs4acl/acl_nfs4_inheritance.c

```diff
@@ -63,14 +63,32 @@ _print_diff_aces(struct nfs4_ace *a, struct nfs4_ace *b)
 	fprintf(stderr, "\n");
 }
 
+bool
+ace_is_equal(struct nfs4_ace *entrya, struct nfs4_ace *entryb)
+{
+	if ((entrya->type != entryb->type) ||
+	    (entrya->whotype != entryb->whotype) ||
+	    (entrya->flag != entryb->flag) ||
+	    (entrya->access_mask != entryb->access_mask) ||
+	    ((strcmp(entrya->who, entryb->who) != 0))) {
+#ifdef NFS4_DEBUG
```

Comment:
I'm not seeing this build time option anywhere? Maybe leftover or a future addition?